api/nodes/serializers.py

```diff
@@ -328,7 +328,8 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
 
     draft_registrations = HideIfRegistration(RelationshipField(
         related_view='nodes:node-draft-registrations',
-        related_view_kwargs={'node_id': '<_id>'}
+        related_view_kwargs={'node_id': '<_id>'},
+        related_meta={'count': 'get_draft_registration_count'}
```

Comment:
I forget: does this only happen when the `?related_count=1` query param is present?

Reply:
Or when listing specific relationships, e.g: `?related_counts=draft_registrations`.

Reply:
Oh ok, thanks.